server/app/controllers/NewTask.scala

```diff
@@ -39,13 +43,16 @@ class NewTask @Inject()(executor: DynamicSuiteExecutor, dao: Dao, val messagesAp
       f => {
         val futureResponse = for {
           _ <- Future(StringBuilderRunner(executor(f.referenceSolution, f.suite))).check
-          db <- dao.addTask(Task(scalaClass, f.taskDescription, f.solutionTemplate, f.referenceSolution, f.suite))
+          db <- dao.addTask(
+            Task(new Timestamp(System.currentTimeMillis),
```

Comment:
Let's put year as int in db :-)
It is strange to keep time stamp just for 1st of January. Or maybe we need change concept from year to creation date field